### PR TITLE
Branch iky

### PR DESCRIPTION
**Pull request by jafidiesel**

### CODE REVIEW

File: src/Components/App/styles.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        },
+
+        flex_container: {
+            display: 'flex',
+            flexDirection: 'column',
+            width: '90vw',
+            height: '90vh',
+        },
+        
+        nav_container: {
```
Comment (jafidiesel) [inline src/Components/App/styles.ts:22]: remove this class

File: src/Components/Body/styles.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+    },
+
+    label: {
+        display: 'flex',
+        fontSize: 'large',
+        fontWeight: 'bold',
+        margin: '5px',
```
Comment (jafidiesel) [inline src/Components/Body/styles.ts:29]: use number instead of string
like this margin: 5

File: src/Components/Body/styles.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    },
+
+    select: {
+        fontSize: 'large',
+        borderRadius: '5px',
+        textAlign: 'center',
+    },
+
+    options: {
+
```
Comment (jafidiesel) [inline src/Components/Body/styles.ts:40]: remove this space

File: src/Components/Body/styles.ts [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    select: {
+        fontSize: 'large',
+        borderRadius: '5px',
+        textAlign: 'center',
+    },
+
+    options: {
+        
+        border: '2 px solid black',
```
Comment (jafidiesel) [inline src/Components/Body/styles.ts:41]: use 2px instead of 2 px

File: src/Components/Footer/Footer.tsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import {css} from 'aphrodite';
+import styles from './styles';
+
+const Footer = () => {
+    return (
+      <div id='item_footer' className={css(styles.footer)}>
+        ProjectGreen - Copywright© 2024 - Powered by
+          <a className={css(styles.footer_link)} href='https://github.com/green-devs-test'> Mancogamers</a>
```
Comment (jafidiesel) [inline src/Components/Footer/Footer.tsx:8]: change this to 
<a className={css(styles.footer_link)} href='https://github.com/green-devs-test'> Green Dev tests</a>

File: src/Components/Icon/Icon.tsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import { css } from 'aphrodite';
+import styles from './styles';
+import { FaBars } from "react-icons/fa";
+
+export const Icon = () => {
+    return (
+        <FaBars className={css(styles.iconito)} />
```
Comment (jafidiesel) [inline src/Components/Icon/Icon.tsx:7]: change `styles.iconito` to `styles.icon`

File: src/Components/Sidebar/styles.ts [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    sidebar: {
+        visibility: 'visible',
+        backgroundColor: '#c2eaba',
+        height: '100%',
+        width: '20vw',
+        display: 'flex',
+        flexDirection: 'column',
+        boxSizing: 'border-box',
+        padding: 7,
+        zIndex: 5,
```
Comment (jafidiesel) [inline src/Components/Sidebar/styles.ts:14]: remove this

File: src/Components/App/styles.ts (hunk 1 above)
Comment (serviceace) [reply src/Components/App/styles.ts]: removed

File: src/Components/Body/styles.ts (hunk 2 above)
Comment (serviceace) [reply src/Components/Body/styles.ts]: done

File: src/Components/Body/styles.ts (hunk 3 above)
Comment (serviceace) [reply src/Components/Body/styles.ts]: removed

File: src/Components/Body/styles.ts (hunk 4 above)
Comment (serviceace) [reply src/Components/Body/styles.ts]: used

File: src/Components/Footer/Footer.tsx (hunk 5 above)
Comment (serviceace) [reply src/Components/Footer/Footer.tsx]: Name changed

File: src/Components/Icon/Icon.tsx (hunk 6 above)
Comment (serviceace) [reply src/Components/Icon/Icon.tsx]: Stile name changed

File: src/Components/Sidebar/styles.ts (hunk 7 above)
Comment (serviceace) [reply src/Components/Sidebar/styles.ts]: removed

File: src/Components/TopBar/TopBar.tsx [hunk 8]
```diff
@@ -1,7 +1,20 @@
...
+import { css } from 'aphrodite';
+import Icon from '../Icon/Icon';
+
+
+interface TopBarProps{
+  toggleSidebar(): void
+}
+
+const TopBar = (props: TopBarProps) => {
+  // TODO: show and hide menu button with css according to media query
```
[the remark this answers is not in the document]
Comment (serviceace) [reply src/Components/TopBar/TopBar.tsx]: Comment removed

**Pull request merged**